provider/go.mod

```diff
@@ -1,9 +1,211 @@
 module github.com/pulumi/pulumi-eks/provider
 
-go 1.16
+go 1.20
 
 require (
 	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi/pkg/v3 v3.64.0
-	github.com/pulumi/pulumi/sdk/v3 v3.64.0
+	github.com/pulumi/pulumi/pkg/v3 v3.81.0
```

Comment:
I see this update is part of this PR, awesome. 

Are we merging this soon-ish? Note also that master branch stopped compiling.

Reply:
Yes. For some definition of soon-ish :)